mods/cnc/maps/nod05/map.yaml

```diff
@@ -0,0 +1,543 @@
+		Location: 52,56
+		Owner: Neutral
+	UnitsRallyVehicle: waypoint
+		Location: 51,49
+		Owner: Neutral
+	UnitsRallyRocket: waypoint
+		Location: 49,49
+		Owner: Neutral
+	UnitsRallyGunner: waypoint
+		Location: 53,49
+		Owner: Neutral
+	UnitsRallyMCV: waypoint
+		Location: 51,52
+		Owner: Neutral
+	Tower1: gtwr
+		Location: 28,21
+		Owner: GDI
+	Tower2: gtwr
+		Location: 23,21
+		Owner: GDI
+	Radar: hq
+		Location: 21,14
+		Owner: GDI
+	Silo1: silo
+		Location: 16,20
+		Owner: GDI
+	Silo2: silo
+		Location: 16,18
+		Owner: GDI
+	Silo3: silo
+		Location: 16,16
+		Owner: GDI
+	Refinery: proc
+		Location: 18,17
+		Owner: GDI
+	Barracks: pyle
+		Location: 25,18
+		Owner: GDI
+	Plant1: nuke
+		Location: 21,17
+		Owner: GDI
+	Plant2: nuke
+		Location: 23,17
+		Owner: GDI
+	Yard: fact
+		Location: 18,14
+		Owner: GDI
+	Factory: weap
+		Location: 26,13
+		Owner: GDI
+
+Smudges:
+	cr1 46,48 0:
+
+Rules:
+	Player:
+		-ConquestVictoryConditions:
+		MissionObjectives:
+			EarlyGameOver: true
+	World:
+		-CrateSpawner:
+		-SpawnMPUnits:
+		-MPStartLocations:
+		ObjectivesPanel:
+			PanelName: MISSION_OBJECTIVES
+		LuaScript:
+			Scripts: nod05.lua
+	^Vehicle:
+		Tooltip:
+			GenericVisibility: Enemy
+			ShowOwnerRow: false
+	^Tank:
+		Tooltip:
+			GenericVisibility: Enemy
+			ShowOwnerRow: false
+	^Infantry:
+		Tooltip:
+			GenericVisibility: Enemy
+			ShowOwnerRow: false
+	^Plane:
+		Tooltip:
+			GenericVisibility: Enemy
+			ShowOwnerRow: false
+	^Ship:
+		Tooltip:
+			GenericVisibility: Enemy
+			ShowOwnerRow: false
+	^Building:
+		Tooltip:
+			GenericVisibility: Enemy
+			ShowOwnerRow: false
+	NUK2:
+		Buildable:
+			Prerequisites: ~disabled
+	GUN:
+		Buildable:
+			Prerequisites: ~disabled
+	CYCL:
+		Buildable:
+			Prerequisites: ~disabled
+	FIX:
+		Buildable:
+			Prerequisites: ~disabled
+	HPAD:
+		Buildable:
+			Prerequisites: ~disabled
+	OBLI:
+		Buildable:
+			Prerequisites: ~disabled
+	BRIK:
+		Buildable:
+			Prerequisites: ~disabled
+	TMPL:
+		Buildable:
+			Prerequisites: ~disabled
+	E5:
+		Buildable:
+			Prerequisites: ~disabled
+	RMBO:
+		Buildable:
+			Prerequisites: ~disabled
+	MLRS:
+		Buildable:
+			Prerequisites: ~disabled
+	MCV:
+		Buildable:
+			Prerequisites: ~disabled
+	LST:
+		Buildable:
+			Prerequisites: ~disabled
+	C17:
+		Buildable:
+			Prerequisites: ~disabled
+	GTWR:
+		Buildable:
+			Prerequisites: ~disabled
+	WEAP:
+		Buildable:
+			Prerequisites: ~disabled
+	EYE:
+		Buildable:
+			Prerequisites: ~disabled
+	ATWR:
+		Buildable:
+			Prerequisites: ~disabled
+	HARV:
+		Buildable:
+			Prerequisites: ~disabled
+	FTNK:
+		Buildable:
+			Prerequisites: ~disabled
+	STNK:
+		Buildable:
+			Prerequisites: ~disabled
+	ARTY:
+		Buildable:
+			Prerequisites: ~disabled
+	E3:
+		Buildable:
+			Prerequisites: ~disabled
+	MTNK:
+		Buildable:
+			Prerequisites: ~disabled
+	HTNK:
+		Buildable:
+			Prerequisites: ~disabled
+	ORCA:
+		Buildable:
+			Prerequisites: ~disabled
+	MSAM:
+		Buildable:
+			Prerequisites: ~disabled
+	airstrike.proxy:
```

Comment:
Damn, I forgot to remove this. Could you please do it?
(We use the `AirstrikePower` from the `HQ` now.)